src/sdk/models/platformCompare/util.test.ts

```diff
@@ -0,0 +1,19 @@
+import expect = require('expect.js');
+import {PlatformComparisonRuleStatus} from './types';
+import {
+	ifStringArrayExists
+} from './util';
+
+describe('ifStringArrayExists', () => {
+	it(' is Compatiable', () => {
```

Comment:
Preceding space not necessary.

Reply:
fixed. removed leading spaces in `it` description